### PR TITLE
Fixes parts of #1913: Migrate docker tests

### PR DESCRIPTION
**Pull request by NishealJ**
## Description

This PR fixes parts of  #1913 & migrates the docker test from Travis to Github Actions

### Changes

- Added worker for docker test

## How to test
-

### CODE REVIEW

File: src/testing/docker/test-standalone.sh [hunk 1]
```diff
@@ -19,9 +20,9 @@ readonly HOST="$(docker port "${CONTAINER_ID}" 80)"
 curl --silent --location "http://${HOST}/repo/" | grep -q "<title>Getting Started with FOSSology</title>"
 
 #### test copyright is running
-docker exec -it "${CONTAINER_ID}" /usr/local/share/fossology/copyright/agent/copyright -h
+docker-compose exec -T web /usr/local/share/fossology/copyright/agent/copyright -h
```
Comment (GMishx) [inline src/testing/docker/test-standalone.sh:23]: The script is made to test the Docker environment built from single container and not from `docker-compose`. For that, the file `src/testing/docker/test-cluster.sh` is used. Can you please revert the changes to `docker exec`?

File: src/testing/docker/test-standalone.sh [hunk 2]
```diff
@@ -2,7 +2,8 @@
 set -o errexit -o nounset -o xtrace
 
 #### build image
-docker-compose build web
+docker-compose build
+docker-compose up -d
```
Comment (GMishx) [inline src/testing/docker/test-standalone.sh:6]: For the same reason, can you please remove `docker-compose up` and build only the `web` container as before?
The line bellow already starts a container for the tests.

**Review by GMishx — CHANGES_REQUESTED** [review]

**Comment by GMishx** [comment]
Also, can you please change and squash your commits to follow the [contributing guidelines](https://github.com/fossology/fossology/blob/master/CONTRIBUTING.md#git-commit-conventions)?

**Comment by NishealJ** [comment]
Will re-open this PR with fixes.

**Pull request closed without merging**